src/plugins/scriptedwizard/resources/wxwidgets/wizard.script

```diff
@@ -1153,18 +1160,100 @@ function OnGetPrevPage_WxAddLib()
 
 function OnEnter_WxAddLib(fwd)
 {
+    if (fwd)
+    {
+        if (IsDLL) // no linking against any winsock needed
+        {
+            Wizard.CheckCheckbox(_T("chkWxLinkWinSock2"), false);
+            Wizard.EnableWindow(_T("chkWxLinkWinSock2"), false);
+        }
+        else
+        {
+            Wizard.CheckCheckbox(_T("chkWxLinkWinSock2"), IntToBool(ConfigManager.Read(_T("/wx_project_wizard/winsock2"), 0)));
+            Wizard.EnableWindow(_T("chkWxLinkWinSock2"), true);
+        }
+    }
     return true;
 }
 
 function OnLeave_WxAddLib(fwd)
 {
     if (fwd)
     {
+        if (GetCompilerFactory().CompilerInheritsFrom(Wizard.GetCompilerID(), _T("gcc*")))
+        {
+            local tempLibArray = ::wxArrayString();
```

Comment:
This line is useless. The line below it would override both the type and the contents of the object.
I think the more correct version is to remove it and change the line below it to be `local tempLibArray`.

Reply:
Seemed odd to me but as I am not familiar with the language, I just copied the code from `SetupAddlLibs()`, assuming it correct.